config/sets/contao/contao-53.php

```diff
@@ -2,10 +2,14 @@
 
 declare(strict_types=1);
 
+use Contao\CoreBundle\File\ModelMetadataTrait;
 use Contao\CoreBundle\Security\ContaoCorePermissions;
+use Contao\Model\MetadataTrait;
+use Contao\Rector\Rector\ChangeTraitRector;
```

Comment:
this seems to be a leftover?

Reply:
Removed in cbba8286ebf7e9fc02e58c851a3746db7fc72f9b